src/service/passport/index.js

```diff
@@ -14,10 +15,15 @@ const configure = async () => {
     case 'local':
       _passport = await local.configure();
       break;
+    case 'openidconnect':
+      _passport = await oidc.configure();
+      break;
     default:
       throw Error(`uknown authentication type ${type}`);
   }
-  _passport.type = authenticationConfig.type;
```

Comment:
@jescalada - why the change here?

Reply:
@JamieSlome The type is being defined twice, once in each `configure` function and again in this file. It's just a temporary fix since I'm getting rid of it entirely in the multiple auth types PR!